tests/zenko_tests/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:18.04
 
+VOLUME /artifacts
```

Comment:
i am not sure we _want_ to create a volume here: which would create lead to automatically creating "anonymous" volume if run without binding/... We can create a volume bind even if the image does not have such a volume, so IMHO we should just leave the image as ebfore (esp. since the `/artifacts` path is a parameter we specify when running the tests)